master/buildbot/changes/gerritchangesource.py

```diff
@@ -74,6 +75,7 @@ class GerritChangeSourceBase(base.ChangeSource):
     def checkConfig(self,
                     gitBaseURL=None,
                     handled_events=("patchset-created", "ref-updated"),
+                    get_files=False,
```

Comment:
it's a good idea to add new parameters *after* the existing ones: this method signature says that it accepts these parameters as keyword ones as well as positional ones...  i'd even suggest to add `*` before that to ensure that in future this parameter is always a keyword one.

Reply:
Agreed - will fix.